backend/src/main/java/com/woowacourse/friendogly/footprint/domain/Location.java

```diff
@@ -12,32 +14,19 @@
 @AllArgsConstructor
 public class Location {
 
-    private static final int BOUNDARY_RADIUS_METER = 1000;
-
     private double latitude;
     private double longitude;
 
-    public boolean isNear(Location other) {
-        return distance(this.latitude, this.longitude, other.latitude, other.longitude) <= BOUNDARY_RADIUS_METER;
+    public boolean isWithin(Location other, int radius) {
+        return distanceAsMeter(other) <= radius;
     }
 
-    private double distance(double lat1, double lon1, double lat2, double lon2) {
-        double theta = lon1 - lon2;
-        double dist = Math.sin(degToRad(lat1)) * Math.sin(degToRad(lat2)) +
-                      Math.cos(degToRad(lat1)) * Math.cos(degToRad(lat2)) * Math.cos(degToRad(theta));
-
-        dist = Math.acos(dist);
-        dist = radToDeg(dist);
-        dist = dist * 60 * 1.1515 * 1609.344;
-
-        return Math.abs(dist);
-    }
-
-    private double degToRad(double deg) {
-        return (deg * Math.PI / 180.0);
-    }
+    private double distanceAsMeter(Location other) {
```

Comment:
```suggestion
    private double calculateDistanceAsMeter(Location other) {
```
메서드명은 동사로 시작하는게 좋겠습니당

Reply:
동사로 통일!! 👍